src/app/models/AnalysisJob.ts

```diff
@@ -105,6 +105,10 @@ export class AnalysisJob extends AbstractModel implements IAnalysisJob {
   public get viewUrl(): string {
     throw new Error("AnalysisJob viewUrl not implemented.");
   }
+
+  public toString(): string {
+    return this.name;
```

Comment:
If the `id` was important enough to show in the abstract model, it is probably useful here too